pkg/service/catalog.go

```diff
@@ -39,9 +41,12 @@ type Service interface {
 	ToJSON() ([]byte, error)
 	GetID() string
 	GetName() string
+	GetBindable() bool
```

Comment:
Can we change this to `IsBindable()`? That seems more conventional when you're asking a yes/no question and expecting a boolean response.